tests/data/hashes.csv

```diff
@@ -1,4 +1,4 @@
-dummy_author/agents/dummy_aea,QmR4SuPKbe4qjgF5DVwVwHPm5eFqCx83sdavs4QEXgF1mR
+dummy_author/agents/dummy_aea,QmVjFRYLEwTF9zK2g4sDTvb3Djo2We1QFiNUHT6EJedgsx

```

Comment:
this is likely due to symlinks. `follow_symlinks=True` didn't solve the difference.

Reply:
Concerning. We need to understand why before we merge this.